backend/app.js

```diff
@@ -3,6 +3,11 @@ const app = express();
 app.get('/', function (req, res) {
   res.send('Hello World!');
 });
-app.listen(3000, () => {
+
+app.listen(3001, () => {
```

Comment:
[In Yasmins PR ](https://github.com/ArnonEilat/kueen-b/pull/38/files) she changes the backend port to be 5000, and the frontend to be 3001.